Rakefile

```diff
@@ -61,9 +61,9 @@ namespace :arm do
 
         if md[:spec_uri].end_with?('.json')
           command = "#{ar_base_command} -i #{md[:spec_uri]} -pv #{md[:version]} -n #{md[:ns]} -pn #{md[:pn].nil? ? dir : md[:pn]} -g Azure.Ruby -o lib"
```

Comment:
is the comment in the if valid at this point? if it's not, either we remove the `if` or we update the command in case we want to generate from a .json directly?